manifests/init.pp

```diff
@@ -197,10 +197,8 @@
     fail("start_delay requires at least Monit 5.0. Detected version is <${monit_version_real}>.")
   }
 
-  anchor { "${module_name}::begin": }
```

Comment:
If you drop the anchor (which is fine), I'd add `contain`. So:

```puppet
contain "${module_name}::install"
contain "${module_name}::config"
contain "${module_name}::service"
contain "${module_name}::firewall"

Class["${module_name}::install"] -> Class["${module_name}::config"] ~> Class["${module_name}::service"] -> Class["${module_name}::firewall"]
```

That way the classes are still contained inside of `Class['monit']`. The `anchor` pattern was always a hack/workaround to achieve what `contain` natively does.

Reply:
alright, will do